### ISSUE TITLE
review exported identifiers

### ISSUE THREAD

**Issue by treinen** [colis-anr/morbig#29]
Just a TODO, not a bug: we should review what precisely is exported in the library. For instance, we probably do  not want to export all of CSTHelpers.

**Comment by Niols** [comment]
You're right. We should also check that it is exported correctly (makes me think of #22). About CSTHelpers, we probably just need a nice `.mli` file (it will also be important for documentation).

**Comment by treinen** [comment]
we should also export a function that allows to output a CST in various formats to an out_channel. This would be useful for tools that use the morbig library and implement various transformations on a CST.

**Comment by yurug** [comment]
I propose to have a single entry point for the library by enriching the interface Libmorbig.API with
- the contents of CST
- I/O functions
Any other ideas for relevant public functions? (Remember that smallest API are the best.)

**Comment by Niols** [comment]
The whole CST in the API?

Also, if the goal is to have the smallest API possible, maybe just a parsing function that works on channels would be good. So that everyone can rewrite the others? I, personally, like have things like `parse_file`, `parse_channel` and `parse_string`.

Also, an entry point to parse as much as possible (a whole file for instance) and an "interactive" entry point that parses only one complete_command (taking as input a map of aliases, probably)?

**Comment by yurug** [comment]
`parse_string` is implemented by bc9e915.

`parse_channel` and `interactive_parse` will not happen in 0.9 because they are not immediate to implement because of lexing backtracking.
